### PR TITLE
Transalte to spanish messages

### PR DESCRIPTION
**Pull request by chernandezq**

### CODE REVIEW

**Comment by hameno** [comment]
Please read http://developer.android.com/guide/topics/resources/localization.html and update your PR accordingly. Also, there is a typo in the commit summary.

**Comment by chernandezq** [comment]
I think thats ok

File: library/res/values/strings.xml [hunk 1]
```diff
@@ -48,4 +48,4 @@
     <string name="formvalidations_not_in_range">Value should be between %1$d and %2$d</string>
     <string name="formvalidations_not_email">Value is not valid email address</string>
     <string name="formvalidations_not_alphanumeric">Value is not alphanumeric</string>
-</resources>
+</resources>
```
Comment (hameno) [inline library/res/values/strings.xml]: revert this change

File: library/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  ~     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~     IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+  ~     FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+  ~     COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+  ~     IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+  ~     CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+  -->
+
+<resources>
+    <string name="formvalidacion_numerica">Están permitidos solo números</string>
```
Comment (hameno) [inline library/res/values-es/strings.xml]: keys are not to be translated

**Comment by hameno** [comment]
Looks better. Please fix the remaining issues and squash your commits into one.

**Comment by chernandezq** [comment]
Ok, the string name="" is in english ?

**Comment by hameno** [comment]
Yes, the value of "name" is the key which identifies a translation. It would make little sense to translate this common key.

**Comment by chernandezq** [comment]
Ok, already

**Comment by hameno** [comment]
Okay, now please squash your changes and fix the typo commit message: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#squash-your-changes

**Comment by chernandezq** [comment]
I have to create branch new ?

**Comment by hameno** [comment]
No, please read my link, it explains every step.

**Comment by chernandezq** [comment]
I don´t understand, You cant help me ?

**Comment by hameno** [comment]
Just read the article, it is not that hard...
